packages/terra-table/docs/guides/SingleSelectTable.md

```diff
@@ -0,0 +1,128 @@
+```jsx
+  handleRowSelection(event, metaData) {
+    event.preventDefault();
+    if (this.state.selectedKey !== metaData.key) {
+      this.setState({ selectedKey: metaData.key });
+    }
+  }
+```
+Setting state will trigger another render. So in the render method we need generate our table rows with the updated isSelected and isSelectable props. Each item needs a unique key, not necessarily associated to our own key, but it works as well. The table renders flat, so keys need to be unique even if they are placed within sections.
+[React List & Key Documentation](https://reactjs.org/docs/lists-and-keys.html)
+```jsx
+  createTableRow(rowData) {
+    return (
+      <Row
+        key={itemData.key}
+      >
+        {createCellsForRow(rowData.cells)}
+      </Row>
+    );
+  }
+```
+Next we need to set up our metaData object with our key value, and attach the "onSelect" to our handler.
+```jsx
+  createTableRow(rowData) {
+    return (
+      <Row
+        key={itemData.key}
+        metaData={{ key: itemData.key }}
+        onSelect={this.handleRowSelection}
+      >
+        {createCellsForRow(rowData.cells)}
+      </Row>
+    );
+  }
+```
+For the single select table we set "isSelectable" for all items.
+```jsx
+  createTableRow(rowData) {
+    return (
+      <Row
+        key={itemData.key}
+        isSelectable
+        metaData={{ key: itemData.key }}
+        onSelect={this.handleRowSelection}
+      >
+        {createCellsForRow(rowData.cells)}
+      </Row>
+    );
+  }
+```
+Next we need to check if the item matches the selectedKey in state.
+```jsx
+  createTableRow(rowData) {
+    return (
+      <Row
+        key={itemData.key}
+        isSelectable
+        isSelected={this.state.selectedKey === itemData.key}
+        metaData={{ key: itemData.key }}
+        onSelect={this.handleRowSelection}
+      >
+        {createCellsForRow(rowData.cells)}
+      </Row>
+    );
+  }
+```
+We can then stub out our static elements
+```jsx
+  const createCell = cell => (
+    <Cell key={cell.key}>
+      <Placeholder title={cell.title} style={{ height: '50px', padding: '0' }} />
+    </Cell>
+  );
+
+  const createCellsForRow = cells => cells.map(cell => createCell(cell));
+```
+Finally we can implement a method to loop through our data and create the table row with our methods and call it from our render method. 
+```jsx
+  createTableRows(data) {
+    return data.map(childItem => this.createTableRow(childItem));
+  }
+
+  render() {
+    return (
+      <Table
+        header={
+          <Header>
+            <HeaderCell>Column 0</HeaderCell>
+            <HeaderCell>Column 1</HeaderCell>
+            <HeaderCell>Column 2</HeaderCell>
+          </Header> 
+        }
+      >
+        {this.createTableRows(mockData)}
+      </Table>
+    );
+  }
+}
+  ```
+  Using these steps we get the following example:
```

Comment:
Oops missing full example

Reply:
The example is built with the the doc template.